build-tools/src/main/kotlin/org/jetbrains/kotlin/CompileToBitcode.kt

```diff
@@ -46,11 +46,45 @@ open class CompileToBitcode @Inject constructor(@InputDirectory val srcRoot: Fil
 
     private val targetDir by lazy { File(project.buildDir, target) }
 
-    private val objDir by lazy { File(targetDir, folderName) }
+    val objDir by lazy { File(targetDir, folderName) }
```

Comment:
I don't like exposing internals here and in `ExecClang`, so that we can scavenge them and assemble a compilation database. But I don't have any better ideas just yet.